apps/lib/publisher_agent/config.go

```diff
@@ -19,25 +19,27 @@ const DefaultClockUpdatePeriod = "500ms"
 const DefaultDeltaUpdatePeriod = "10ms"
 const DefaultChangeThresholdPercent = 0.1
 const DefaultStorkRegistryRefreshInterval = "10m"
-const DefaultStorkRegistryBaseUrl = "https://rest.jp.stork-oracle.network"
+const DefaultPublisherMetadataRefreshInterval = "1h"
+const DefaultStorkRestApiBaseUrl = "https://rest.jp.stork-oracle.network"
 const DefaultBrokerReconnectDelay = "5s"
 const DefaultPullBasedReconnectDelay = "5s"
 const DefaultPullBasedReadTimeout = "10s"
 
 type ConfigFile struct {
-	SignatureTypes                 []signer.SignatureType
-	ClockPeriod                    string
-	DeltaCheckPeriod               string
-	ChangeThresholdPercent         float64 // 0-100
-	StorkRegistryBaseUrl           string
-	StorkRegistryRefreshInterval   string
-	BrokerReconnectDelay           string
-	PullBasedWsUrl                 string
-	PullBasedWsSubscriptionRequest string
-	PullBasedWsReconnectDelay      string
-	PullBasedWsReadTimeout         string
-	SignEveryUpdate                bool
-	IncomingWsPort                 int
+	SignatureTypes                   []signer.SignatureType
+	ClockPeriod                      string
+	DeltaCheckPeriod                 string
+	ChangeThresholdPercent           float64 // 0-100
+	StorkRestApiBaseUrl              string
```

Comment:
renamed `StorkRegistryBaseUrl` -> `StorkRestApiBaseUrl` so we can use it for both the registry and metadata reporting.